template/cordova/prebuild.js

```diff
@@ -50,7 +50,9 @@ module.exports = function patch (platform) {
     // 1. Find start page path in appxmanifest
     var AppxManifest = require('./lib/AppxManifest');
     var appxmanifest = AppxManifest.get(path.join(__dirname, '..', appxmanifestMap[platform]));
-    var startPage = url.parse(appxmanifest.getApplication().getStartPage());
+
+    // @todo Use 'url.URL' constructor instead
```

Comment:
?

Reply:
ESLint Notice
```
55:21  error  'url.parse' was deprecated since v11.0.0. Use 'url.URL' constructor instead  node/no-deprecated-api
```

Reply:
I would favor fixing this one before making the new release, if possible.

Reply:
The ESLint notice is for Node 11+, when it becomes deprecated and which we do not support yet.

Per documentations, changes would be as follows:
```
var startPage = new URL(appxmanifest.getApplication().getStartPage());
```
or
```
var startPage = new url.URL(appxmanifest.getApplication().getStartPage());
```

`appxmanifest.getApplication().getStartPage()` pulls the start page from `package.*.appxmanifest`, which is `www/index.html`.

Here is the output of the current usage `url.parse("www/index.html")`:
```
Url {
  protocol: null,
  slashes: null,
  auth: null,
  host: null,
  port: null,
  hostname: null,
  hash: null,
  search: null,
  query: null,
  pathname: 'www/index.html',
  path: 'www/index.html',
  href: 'www/index.html' }
```

Now lets take a look at `new URL("www/index.html")` or `new url.URL("www/index.html")`

**Node 6**
with `new URL("www/index.html")`:
```
ReferenceError: URL is not defined
```
with `new url.URL("www/index.html")`:
```
TypeError: Invalid URL: www/index.html
```

**Node 8**
`new URL("www/index.html")`:
```
ReferenceError: URL is not defined
```
`new url.URL("www/index.html")`:
```
TypeError [ERR_INVALID_URL]: Invalid URL: www/index.html
  at onParseError (internal/url.js:219:17)
  at parse (internal/url.js:228:3)
  at new URL (internal/url.js:311:5)
```

**Node 10**
`new URL("www/index.html")`
and
`new url.URL("www/index.html")`:
```
{ TypeError [ERR_INVALID_URL]: Invalid URL: www/index.html
  at onParseError (internal/url.js:240:17)
  at parse (internal/url.js:249:3)
  at new URL (internal/url.js:324:5) input: 'www/index.html' }
```

Since we don't support Node 11, I am hoping we can focus on this after the release.